### DISCUSSION TITLE
Unable to use UUID

### DISCUSSION THREAD

**Discussion by paramwork** [Kyon147/laravel-shopify#340] [General]
I have set everything up correctly, but when I test the store for authentication, I get the following error:

```SQLSTATE[01000]: Warning: 1265 Data truncated for column 'id' at row 1 (Connection: mysql, SQL: insert into `users` (`name`, `password`, `email`, `id`, `updated_at`, `created_at`) values (list-test.myshopify.com, , shop@list-test.myshopify.com, 9cd7cf81-e615-49b7-ba40-7bf22ddacdbb, 2024-08-24 13:45:32, 2024-08-24 13:45:32))```

What can I do to fix this? I tried changing the `id` in the `users` table to `string`, but then I received a foreign key mismatch error.

How can I use UUID with this package? Please help me solve this error.

**Comment by Kyon147** [comment]
Why would you want to use a uuid instead of just leaving it as it is?
